### PR TITLE
Removes headpat gloves

### PR DESCRIPTION
**Pull request by Ty-Omaha**
Some admins are lagging from this and the logs are being spammed

We really don't need this much spam in game either way

🆑 
del: Headpat gloves
/ 🆑

### CODE REVIEW

**Comment by Drakeven** [comment]
We hardly knew ye...

**Comment by AffectedArc07** [comment]
Or just make them not admin log, or increase the cooldown


***imPrOve donT RemOVE***

**Comment by Ty-Omaha** [comment]
@AffectedArc07 
lots of things get removed instead of improved

**Comment by Lunaimon** [comment]
How did these get in lmao

**Comment by Ty-Omaha** [comment]
@AffectedArc07 updated to keep the refactor

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -161,16 +161,3 @@
 	if(M.a_intent in accepted_intents)
 		M.changeNext_move(click_speed_modifier)
 	.= FALSE
-
-/obj/item/clothing/gloves/fingerless/rapid/admin
```
Comment (AffectedArc07) [inline code/modules/clothing/gloves/miscellaneous.dm]: You sure you dont want to keep these as well?

**Comment by FreeStylaLT** [comment]
At first I thought we should just OOCly remind spammers not to spam excessively.
Then I remembered that spam is the entire purpose of these items. They probably shouldn't be publicly accessible like that, at least.

File: code/modules/clothing/gloves/miscellaneous.dm (hunk 1 above)
Comment (Ty-Omaha) [reply code/modules/clothing/gloves/miscellaneous.dm]: eh sure

**Comment by Qwertytoforty** [comment]
Honestly, gloves are annoying, probably could be abused by shaking people up to remove stuns fast, so it is better to remove them or keep them admin only.

**Comment by Spacemanspark** [comment]
I'd be fine with making it admin only.

**Comment by Ty-Omaha** [comment]
@Spacemanspark the admin only version is kept

**Comment by datlo** [comment]
just remove the gloves from the vendor instead

**Comment by Fox-McCloud** [comment]
There's several different ways this could be improved; kneejerk removing it when it's very modular is silly.

Instead of out and out removing it, you could:

- Make the cooldown higher
- Have it print a message to people in a small radius instead of it being a visible message
- Move it from the vendor (where people are exploiting the heck out of decon and rebuilt vending machines to get more) to somewhere else harder to get (like an arcade thing)
- Fix logging on help intent so it doesn't show up for everything
- Any combination of the above.

This isn't some snowflake code that's hard to modify; I had this this way so it was *easy* to modify.

**Comment by Ty-Omaha** [comment]
>Make the cooldown higher

What's the point of it then? Slightly faster hugs?

>Have it print a message to people in a small radius instead of it being a visible message

Those people in a small radius will still get spammed

>Move it from the vendor (where people are exploiting the heck out of decon and rebuilt vending machines to get more) to somewhere else harder to get (like an arcade thing)

I don't see the purpose of having an item just to circumvent spam protection, arcades can be exploited and even if someone wins it then we are at the same issue of having spam

>Fix logging on help intent so it doesn't show up for everything

As above



What value does this add to the game other than spamming hugs?

**Comment by Fox-McCloud** [comment]
>What's the point of it then? Slightly faster hugs?

What's the point of gloves of the north star? Slightly faster punching? 

There's a world of difference between "reduced cooldown" and "literally none at all; as fast as you can click"

>Those people in a small radius will still get spammed

The vast bulk of the complaint as about logging and causing it to be shown to *everyone* on the screen.

>arcades can be exploited and even if someone wins it then we are at the same issue of having spam

The problem has been 3-4+ people having them and all using them, not to mention the *literally non-existent click cooldown*.

**Comment by Ty-Omaha** [comment]
>What's the point of gloves of the north star? Slightly faster punching?

Yes, to give antags another way of killing people.

**Comment by variableundefined** [comment]
Closing in favour of fixing the cooldown by using headpat's parent's cooldown.

**Pull request closed without merging**